components/core/anchor.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components'
+
+export default styled.a`
```

Comment:
why is this file called anchor? isn't it a link not an anchor? 

https://www.w3schools.com/tags/tag_a.asp